config/src/config/quorum_store_config.rs

```diff
@@ -38,6 +39,7 @@ impl Default for QuorumStoreBackPressureConfig {
             decrease_duration_ms: 1000,
             increase_duration_ms: 1000,
             decrease_fraction: 0.5,
+            increase_fraction: 1.5,
```

Comment:
I don't see this being used anywhere - are you missing some changes?

Reply:
Oops. Not sure what happend. I was using the increase_fraction in batch_generator. Pushed the changes.